src/main/resources/en/device.yml

```
@@ -1,7 +1,163 @@
+        - iPhone 4
+        - iPhone 4S
+        - iPhone 5
+        - iPhone 5C
+        - iPhone 5S
+        - iPhone 6 / 6 Plus
+        - iPhone 6S / 6S Plus
+        - iPhone SE
+        - iPhone 7 / 7 Plus
+        - iPhone 8 / 8 Plus
+        - iPhone X
+        - iPhone XR
+        - iPhone XS
+        - iPhone XS Max
+        - iPhone 11
+        - iPhone 11 Pro
+        - iPhone 11 Pro Max
+        - iPhone SE (2nd gen)
+        - iPhone 12
+        - iPhone 12 mini
+        - iPhone 12 Pro
+        - iPhone 12 Pro Max
+        - iPhone 13
+        - iPhone 13 mini
+        - iPhone 13 Pro
+        - iPhone 13 Pro Max
+        - iPhone SE (3rd gen)
+        - iPhone 14
+        - iPhone 14 Plus
+        - iPhone 14 Pro
+        - iPhone 14 Pro Max
+        - OnePlus 6T
+        - OnePlus 6
+        - OnePlus 5T
+        - OnePlus 5
+        - OnePlus 3T
+        - OnePlus 3
+        - OnePlus 2
+        - OnePlus One
+        - Samsung Galaxy S9
+        - Samsung Galaxy S8 Plus
+        - Samsung Galaxy S8
+        - Samsung Galaxy S7 Edge
+        - Samsung Galaxy S7
+        - Samsung Galaxy S6 Edge
+        - Samsung Galaxy S5
+        - Samsung Galaxy S4
+        - Samsung Galaxy S4 Mini
+        - Samsung Galaxy S3
+        - Samsung Galaxy S3 Mini
+        - Samsung Galaxy S2
+        - Samsung Galaxy S1
+        - Huawei Mate 20
+        - Huawei Mate 10 Pro
+        - Huawei P20
+        - Huawei P20 Pro
+        - Huawei Mate 20 Pro
+        - Huawei P10 Plus
+        - Huawei P10
+        - Huawei Mate 20 Lite
+        - Huawei Mate 9
+        - Huawei P Smart
+        - Google Pixel
+        - Google Pixel XL
+        - Google Pixel 2
+        - Google Pixel 2 XL
+        - Google Pixel 3
+        - Google Pixel 3 XL
+        - Google Pixel 3a
+        - Google Pixel 3a XL
+        - Google Pixel 4
+        - Google Pixel 4 XL
+        - Google Pixel 4a
+        - Google Pixel 5
+        - Google Pixel 5a 5G
+        - Google Pixel 6
+        - Google Pixel 6 Pro
+        - Google Pixel 6a
+        - Google Pixel 7
+        - Google Pixel 7 Pro
+        - Xiaomi Redmi Note 6 Pro
+        - Xiaomi Mi 8 Pro
+        - Xiaomi Mi 8 Lite
+        - Xiaomi Mi 8
+        - Xiaomi Mi 8 SE
+        - Xiaomi Pocophone F1
+      platform:
+        - Android OS
+        - webOS
+        - iOS
+        - BlackBerry
+        - Danger OS
+        - Android
+        - Firefox OS
+        - Ubuntu Touch
+        - Windows Phone
+        - Windows 8
+        - Windows RT
+        - Windows 8.1
+        - Windows 10
+        - Windows 10 Mobile
+        - Windows Phone
+      manufacturer:
+        - ASUS
+        - Acer
+        - Amazon
+        - Apple
+        - BlackBerry
+        - Dell
+        - HP
+        - HTC
+        - Huawei
+        - Lenovo
+        - Microsoft
+        - Motorola
+        - Nokia
+        - OnePlus
+        - Oppo
+        - Panasonic
+        - Philips
+        - Plum
+        - QMobile
+        - Sharp
```

```suggestion
        - Samsung
        - Sharp
```

Not sure how this keeps being overlooked but they’re a (maybe the) major Android brand in North America

---

Thanks. Added.